cmd/cluster.go

```diff
@@ -12,13 +12,18 @@ func setupClusterCommand() *cobra.Command {
 		Use:   "up",
 		Short: "Boot up the testing cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := cluster.BootUp()
+			d, err := cmd.Flags().GetBool("d")
+			if err != nil {
+				return err
```

Comment:
Please wrap the error. See: https://github.com/elastic/elastic-package/blob/master/cmd/testrunner.go#L52